components/net/bluetooth_thread.rs

```diff
@@ -48,6 +52,48 @@ macro_rules! return_if_cached(
     );
 );
 
+macro_rules! check_blacklist {
+    ($blacklist:expr, $uuid:expr, $func:expr, $sender:expr) => (
```

Comment:
Can we use `self.bluetooth_blacklist` in the macro? Because the `blacklist` var is always that, so it shouldn't be a param.